test/compilable/showSafe.d

```diff
@@ -0,0 +1,59 @@
+// REQUIRED_ARGS: -transition=safe
+/*
+TEST_OUTPUT:
+---
+compilable/showSafe.d(14): pointers is un-@safe but not marked as @system
```

Comment:
Can you add an example with a non-annotated `@safe` function ?

Reply:
See `implicitSafe`